### DISCUSSION TITLE
Proctoring system

### DISCUSSION THREAD

**Discussion by khushieee** [sanket2000/proctoring#5] [General]
I want to  get help to run it

**Comment by sanket2000** [comment]
Let's discuss this on GitHub itself so the solution can help others in the community too. You can share in detail what is the problem you are facing.

**Reply by khushieee** [reply]
Ohkayy i ran  it do you have its report

**Reply by sanket2000** [reply]
It's already uploaded in this repo.

If you are using this for your course project, I suggest you to enhance on top of this project, instead of directly using this as is. 

It's always a learning experience even of you enhance it in a bare minimum way.

All the best!

**Reply by khushieee** [reply]
I can't see where exactly it is can you provide me here

**Reply by sanket2000** [reply]
`/docs`

Also you could have forked the repo instead to keep it linked to the original for easier updates and contributions!

**Reply by khushieee** [reply]
Can't open the doc

**Reply by sanket2000** [reply]
Can not help if it's not opening for you.
It is already uploaded in pdf format and I can access it without any issues in the `docs/` folder.

**Reply by sanket2000** [reply]
Marking the thread as resolved as I don't see any project related queries, feel free to reopen if you have any thing related.

All the best!
